src/app/waiting/page.tsx

```diff
@@ -1,15 +1,21 @@
 'use client';
 
 import Image from 'next/image';
+import { useEffect, useRef, useState } from 'react';
 import CopyToClipboard from 'react-copy-to-clipboard';
 import SVGLink from '@/styles/icons/link.svg';
-import { useRef } from 'react';
 
 export default function WaitingView() {
+  const [url, setUrl] = useState<string | undefined>(undefined);
```

Comment:
이건 개인적인 코드 스타일일 수 있는데, undefined를 쓰는것 보다 그냥 빈 string을 쓰는것이 어떨까 제안합니다.
```suggestion
const [url, setUrl] = useState<string>("/");
```

undefined를 쓰면 예상치 못한 에러가 발생할 수 있는데 string을 쓰면 에러는 안날것 같기도 해서요.

Reply:
완전 빈 string이 더 나을 것 같아서
 `const [url, setUrl] = useState<string>('');`
 이렇게 수정하였습니다